107/src/main/java/org/ehcache/jsr107/EhcacheCachingProvider.java

```diff
@@ -118,13 +118,14 @@ public CacheManager getCacheManager(URI uri, ClassLoader classLoader, Properties
     return cacheManager;
   }
 
+  @ServiceDependency(services = ManagementRegistry.class)
```

Comment:
Only by renaming `services()` to `value()` but as we have only one annotation parameter, this seems to be the convention. Will do that.